src/game/components/AvailbleActions.tsx

```diff
@@ -0,0 +1,190 @@
+import React, { useState, useCallback, useEffect } from 'react';
+import styled from 'styled-components';
+import update from 'immutability-helper';
+import { DndProvider } from 'react-dnd';
+import { HTML5Backend } from 'react-dnd-html5-backend';
+import { useParams, Link } from 'react-router-dom';
+import Players, { Instruction } from '../consts';
+import ActionsContainer from './ActionsContainer';
+import { DroppableBin } from './DroppableBin';
+
+const Wrapper = styled.div.attrs({ dir: 'rtl' })`
+  grid-template-columns: repeat(2, 1fr);
+  display: grid;
+  flex: 1;
```

Comment:
donno if grid works well together with flex, check if there is an option to get them same result with grid only of flex only